Extensions/autoscroll.js

```diff
@@ -70,41 +76,46 @@ XKit.extensions.autoscroll = new Object({
 			}
 		};
 		$(document).on("click",".auto_scroll_control_step", function(event){
-			// Mark and store the selected speed.
-			controlSteps.removeAttr("selected");
-			$(event.target).attr("selected","true");
-			currentStep = $(event.target).attr("number");
-			XKit.console.add("Scroll index: " + currentStep);
-
-			// Start scrolling
-			var wasScrolling = isScrolling;
-			isScrolling = true;
-			// Function must be started after isScrolling is set
-			// to prevent possible race condition.
-			if(wasScrolling === false)
+			var attr = $(event.target).attr("selected");
+			var ifBool = !$(event.target).is("[selected]");
```

Comment:
this doesn't need to be a local variable, it can go right in the if statement.

also why !.is over !.attr? Off the top of my head I can't think of a situation where they wouldn't be equivalent